src/PreCICE.jl

```diff
@@ -931,16 +931,16 @@ coordinates omitting the mapping. This function is still experimental.
 - `vertexIDs::AbstractArray{Integer}`: IDs of the vertices.
 - `vertexCoordinates::AbstractArray{Float64}`: Coordinates of the vertices and corresponding data values. Shape [N x D] where N = number of vertices and D = number of data dimensions.
 """
-function getMeshVerticesAndIDs(
+function getMeshVertexIDsAndCoordinates(
     meshName::String,
 )::Tuple{AbstractArray{Integer},AbstractArray{Float64}}
-    @warn "The function getMeshVerticesAndIDs is still experimental"
+    @warn "The function getMeshVertexIDsAndCoordinates is still experimental"
 
     _size = getMeshVertexSize(meshName)
     vertexIDs = zeros(Cint, _size)
     vertexCoordinates = zeros(Float64, _size * getMeshDimensions(meshName))
     ccall(
-        (:precicec_getMeshVerticesAndIDs, "libprecice"),
```

Comment:
I think it is because the tests just replicate the Julia function in C and don't actually link it to the precice library.

Reply:
Yes, we test against a mocked interface, and not a real C interface, because we want to control what is returned so that we can test it in the unit testing. Checking against the actual interface is done only via the solverdummy, which does not cover all the API functions.